src/Uno.UI/UI/Xaml/DependencyProperty.cs

```diff
@@ -23,7 +23,7 @@ namespace Windows.UI.Xaml
 	/// Defines a dependency property for a <see cref="DependencyObject"/>.
 	/// </summary>
 	/// <remarks>The properties are attached to the <see cref="IDependencyObject"/> marker interface.</remarks>
-	[DebuggerDisplay("Name={Name}, Type={Type.FullName}, Owner={OwnerType.FullName}, DefaultValue={Metadata.DefaultValue}")]
```

Comment:
did it cause issues ?

Reply:
`DependencyProperty.Metadata` doesn't exist (looks like it moved to `DependencyPropertyDetails`?) so the whole display string was being ignored.